src/OrbitQt/TrackTypeItemModelTest.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QAbstractItemModelTester>
+#include <QModelIndex>
+#include <QSignalSpy>
+#include <QTableView>
+#include <QTest>
+#include <string>
+
+#include "QtUtils/AssertNoQtLogWarnings.h"
+#include "TimeGraphLayout.h"
+#include "TrackManager.h"
+#include "TrackTestData.h"
+#include "TrackTypeItemModel.h"
+
+namespace orbit_qt {
+
+class TrackTypeItemModelTest : public ::testing::Test {
+ public:
+  TrackTypeItemModelTest() : Test() {}
```

Comment:
This is not needed (as it is it's probably even a clang-tidy warning telling you to use `= default`).